flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorSubmissionTest.java

```diff
@@ -555,138 +555,88 @@ public void testFailingScheduleOrUpdateConsumers() throws Exception {
 	}
 
 	// ------------------------------------------------------------------------
-	// Stack trace sample
+	// Back pressure request
 	// ------------------------------------------------------------------------
 
 	/**
-	 * Tests sampling of task stack traces.
+	 * Tests request of task back pressure.
 	 */
-	@Test(timeout = 10000L)
-	@SuppressWarnings("unchecked")
-	public void testRequestStackTraceSample() throws Exception {
-		final ExecutionAttemptID eid = new ExecutionAttemptID();
-		final TaskDeploymentDescriptor tdd = createTestTaskDeploymentDescriptor("test task", eid, BlockingNoOpInvokable.class);
-
-		final int sampleId1 = 112223;
-		final int sampleId2 = 19230;
-		final int sampleId3 = 1337;
-		final int sampleId4 = 44;
+	@Test(timeout = 20000L)
```

Comment:
why we need more timeout here.

Reply:
I am trying to ensure that the output buffer can be exhausted in this timeout.